kiosk/components/PaymentConfirmation.tsx

```diff
@@ -33,7 +33,10 @@ const PaymentConfirmation = (props: PaymentConfirmationProps): JSX.Element => {
       return;
     }
 
-    const personalLinkToWebsite = `${process.env.NEXT_PUBLIC_WEBSITE_URL}/?email=${email}`;
+    // Encode email to preserve any special characters (e.g. +, &)
+    const encodedEmail = encodeURIComponent(email);
```

Comment:
There is still a problem with this solution. The + character is converted to a space instead of being converted to `%2b`.

field input: jel+something@sitecore.net
QR code link: https://jel-pull0516-website.sitecoredemo.com/?email=jel%20something%40sitecore.net
CDP IDENTITY call identity: {"provider":"email","id":"jel something@sitecore.net"}

Reply:
Not sure where it went wrong for you, but getting the correct behavior in all my tests

![image](https://github.com/Sitecore/Sitecore.Demo.Edge/assets/96472119/c7f4c9c1-7908-4786-b529-72e54f3efe54)

![image](https://github.com/Sitecore/Sitecore.Demo.Edge/assets/96472119/990b60bb-bc88-40b1-a2e8-2cb83dddfac8)

Reply:
This [https://github.com/vercel/next.js/issues/11016#issuecomment-778334707](url) appears to be the issue, so currently looking for a workaround.

Reply:
Maybe. I just retested using the portal instance that you started earlier today. I still get `%20` for `+` characters. I am using Microsoft Edge.